packages/headless/src/controllers/core/tab/headless-core-tab.ts

```diff
@@ -1,7 +1,12 @@
 import {BooleanValue, Schema} from '@coveo/bueno';
```

Comment:
A bit out of scope, maybe, but there's an opportunity here to refactor the "isActive" part of the state into a Redux selector as I suggested for the tab manager's `currentTab`.